app/controllers/api/v1/users_controller.rb

```diff
@@ -1,9 +1,26 @@
 class Api::V1::UsersController < ApiController
 
+  swagger_controller :users, 'Users'
+
+  swagger_api :index do
+    summary 'Responds with a list of users'
+    notes 'This lists all users in the system'
+    response :unauthorized
+  end
+
   def index
     @users = User.all
   end
 
+  swagger_api :show do
+    summary 'Responds with a user resource'
+    param :path, :user, :string, :required, "User Id"
```

Comment:
Sorry, no idea - I can't tell from the endpoint and there are no cucumber tests for it.
I just added this documentation so you can see what it is like - I thought the idea was to use "friendly" usernames so I assume it should be /api/users/(username)